### ISSUE TITLE
suppress the announcement of iframe src="about:blank" 

### ISSUE THREAD

**Issue by brennanyoung** [nvaccess/nvda#10273]
# NVDA announces the "about:blank" src attribute of not-yet-loaded iframes. 
I can't see how this is useful to anyone. It's an ungrammatical, machine-readable string, the very opposite of 'meaningful content', which adds unnecessary noise. The announcement happens even if the src attribute is set immediately onload, which is further misinformation about what is going on.

### Describe the solution you'd like
Only announce "about:blank" if no accessible name (i.e. a title or aria-label) for the iframe is provided.

### Describe alternatives you've considered
Something more 'human' and meaningful than the literal announcement of "about:blank"

**Comment by Adriani90** [comment]
cc: @michaelDCurran

**Comment by Adriani90** [comment]
@brennanyoung do you have a minimal test case so we can investigate further?

**Comment by Adriani90** [comment]
Given we don't have a minimal test case and no update info from the initial author upon requuest since many years, I am closing as abandoned. if you can provide the inf requested, please comment and we can reopen.
